portfolio/src/main/java/com/google/sps/servlets/AmazonServlet.java

```diff
@@ -0,0 +1,42 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.LinkedHashMap;
+import java.util.Scanner;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Returns bigfoot data as a JSON object, e.g. {"2017": 52, "2018": 34}] */
+@WebServlet("/amazon-data")
+public class AmazonServlet extends HttpServlet {
+  private LinkedHashMap<String, Float> AmazonData = new LinkedHashMap<>();
```

Comment:
nit, this should start with lowercase, like `amazonData`.  I believe starting with uppercase is reserved for classnames (like LinkedHashMap)

Reply:
addressed in "fixed case of a in amazonData"